build.sh

```diff
@@ -117,14 +117,14 @@ LIBS_ONLY_BUILD=
 SHOULD_EMBED_ICU=0
 ALWAYS_YES=0
 
-if [ -f "/proc/version" ]; then
+UNAME_S=`uname -s`
+if [[ $UNAME_S =~ 'Linux' ]]; then
     OS_LINUX=1
-    PROC_INFO=$(cat /proc/version)
-    if [[ $PROC_INFO =~ 'Ubuntu' || $PROC_INFO =~ 'Debian'
-       || $PROC_INFO =~ 'Linaro' ]]; then
+    PROC_INFO=$(which apt-get)
```

Comment:
Is PROC_INFO still used/does it expect to be a path to apt-get (where it used to the contents of /proc/version)?

Reply:
Good point. See updated IF condition right after that line.